python/tests/test_highlevel.py

```diff
@@ -5491,3 +5491,91 @@ def test_error_if_no_schema(self, table_name):
         ts = msprime.simulate(10)
         with pytest.raises(NotImplementedError):
             getattr(ts, f"{table_name}_metadata")
+
+
+class TestIndividualNodes:
+    def test_basic_individual_nodes(self, tmp_path):
+        # Create a basic tree sequence with two individuals
+        tables = tskit.TableCollection(sequence_length=100)
+        tables.individuals.add_row(flags=0, location=(0, 0), metadata=b"")
+        tables.individuals.add_row(flags=0, location=(0, 0), metadata=b"")
+        tables.nodes.add_row(flags=tskit.NODE_IS_SAMPLE, time=0, individual=0)
+        tables.nodes.add_row(flags=tskit.NODE_IS_SAMPLE, time=0, individual=0)
+        tables.nodes.add_row(flags=tskit.NODE_IS_SAMPLE, time=0, individual=1)
+        tables.nodes.add_row(flags=tskit.NODE_IS_SAMPLE, time=0, individual=1)
+        ts = tables.tree_sequence()
+
+        result = ts.individual_nodes
+        assert result.shape == (2, 2)
+        assert np.array_equal(result, [[0, 1], [2, 3]])
```

Comment:
may as well use ``nt.assert_array_equal``

Reply:
Done in 3873bcbd